### PR TITLE
[bitnami/prometheus-operator] Add Prometheus alert relabel configuration via externally managed secret

### LINKED ISSUE #3197: [bitnami/prometheus-operator] Ability to apply alert_relabel_configs
**Which chart**:
bitnami/prometheus-operator: latest

**Is your feature request related to a problem? Please describe.**
As far as I can see, it is not possible to apply relabeling before alerts are sent to alertmanager.

**Describe the solution you'd like**
Ability to specify alert_relabel_configs in the helm chart values. For example: `prometheus.alert_relabel_configs` or `prometheus.alerting.alert_relabel_configs`.

**Describe alternatives you've considered**

**Additional context**
I am looking to apply alert relabeling before they are sent to alertmanager. I can find the following documentation related to this:
* https://prometheus.io/docs/prometheus/latest/configuration/configuration/#alert_relabel_configs
* https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration
image

Please let me know if this is already possible. As far as I can see, relabeling only applies to the servicemonitor? Unless I understand something incorrectly?

Many Thanks.

### PR DESCRIPTION
**Pull request by dbrennand**
**Description of the change**
This PR implements a new chart value `prometheus.additionalAlertRelabelConfigsExternal`. This value contains 3 sub values:

1. `enable`
2. `name`
3. `key`

This new value allows the user inject additional configuration for Prometheus to use when relabeling alerts before sending them to AlertManager.

I used the following to implement this feature:
* https://github.com/bitnami/charts/blob/master/bitnami/prometheus-operator/templates/prometheus-operator/crd-prometheus.yaml#L80

Mostly based off of PR #3188 

I placed the `prometheus.additionalAlertRelabelConfigsExternal` underneath the other config external value `additionalScrapeConfigsExternal` as they are extremely similar in usage.

**Benefits**
Users will be able to inject custom alert relabeling configuration into Prometheus increasing functionality of the chart.

**Possible drawbacks**
There are no drawbacks that I can think of.

**Applicable issues**

  - fixes #3197 

**Additional information**
I added a "maybe" in one of the comments in the values.yaml files as I am not sure if Prometheus will fail to provision if the secret name or key is entered incorrectly.

**Checklist** 
- [ x ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ x ] Variables are documented in the README.md
- [ x ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [ x ] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by dbrennand** [comment]
Hi @juan131 A lot of the automated checks are failing. 😞

**Comment by dbrennand** [comment]
😕 Not sure where the linting error was. I thought it was the extra space which I fixed, however it's still complaining about the "-" character?
I'm not amazing with Go templating so maybe you can spot where I have made a mistake? 😃

**Comment by juan131** [comment]
I don't see anything wrong in your code.. Indeed I tested it locally and it works as expected... Let me check what's going on here

File: bitnami/prometheus-operator/values-production.yaml [hunk 1]
```diff
@@ -642,6 +642,18 @@ prometheus:
     ##
     # key:
 
+  ## Enable additional Prometheus alert relabel configs that are managed externally to this chart
+  ## Note that the prometheus will fail to provision if the correct secret does not exist.
+  ## ^^ Maybe??
```
Comment (dbrennand) [inline bitnami/prometheus-operator/values-production.yaml:647]: @juan131 Do you know if this is actually the case if the secret is not entered correctly? It behaves the same as `additionalScrapeConfigsExternal` ?

**Comment by juan131** [comment]
I pushed a commit fixing some indentation.. Let's see what happens but it's really weird

**Comment by juan131** [comment]
And it was magically fixed 😮  ...

**Comment by juan131** [comment]
Merging the PR.. I don't know exactly what happened with the CI/CD but it was fixed (Idk how)

**Comment by cblackuk** [comment]
Thanks for that @juan131  - you just made our and somebody else's world, a better place!

**Comment by dbrennand** [comment]
😄 Magics! Love it! Thanks for testing and fixing the indentation issue! ❤️

**Comment by dbrennand** [comment]
Hey @juan131 
When will a new version of the chart be released?
Currently, the version of get is `0.22.3` 😢 

```powershell
helm repo add bitnami https://charts.bitnami.com/bitnami
"bitnami" has been added to your repositories
helm repo update
...
helm search repo prometheus-operator
NAME                            CHART VERSION   APP VERSION     DESCRIPTION
bitnami/prometheus-operator     0.22.3          0.40.0          The Prometheus Operator for Kubernetes provides...
```

**Comment by juan131** [comment]
Hi @dbrennand 

It usually takes a couple of hours, but we have some issues in our pipeline (the team is looking into it) and there are several chart releases that are delayed. Sorry for the inconveniences.

I'll ping you as soon as the version is published.

**Comment by juan131** [comment]
Well.. Indeed it seems yours was a lucky one and it's already published:

```console
$ helm repo update && helm search repo bitnami/prometheus-operator
NAME                       	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/prometheus-operator	0.26.0       	0.40.0     	The Prometheus Operator for Kubernetes provides...
```

**Pull request merged**